src/Grid.php

```diff
@@ -513,6 +513,32 @@ public function addModalAction($button, $title, \Closure $callback, $args = [])
         return $this->getActionButtons()->addModal($button, $title, $callback, $this, $args);
     }
 
+    /**
+     * Similar to addModalAction but apply to a multiple recors selection and display in menu.
+     * When menu item is clicked, modal is displayed with the $title and $callback is executed through
+     * VirtualPage.
+     *
+     * @param string|array|MenuItem $item
+     * @param \Closure(View, array|null): void $callback
+     * @param array                            $args     extra url argument for callback
```

Comment:
```suggestion
     * @param string|array|MenuItem              $item
     * @param \Closure(View, list<string>): void $callback
     * @param array                              $args     extra url argument for callback
```

`$args` are probably not needed - what is the usecase?

also I do not understand why explicit `[]` is needed as `$this->menu->addItem` 2nd arg

Reply:
`[]` removed; not sure for `$args` but `addModalAction` has this and I used it as template.